### PR TITLE
Improve `/show` command output

### PR DESCRIPTION
**Pull request by dector**
## :wrench: Changes

  - Made `/show` command output more human-readable.
  - Updated translations: added new strings to :ukraine: `uk_UA` localization.
  - Extended `.gitignore` file a little.
  - Fixed `pytest` run.

## :white_check_mark: Tests

Some primitive unit tests for `ShowCommand`. But their quality isn't very good. 

## :framed_picture: Result

new `/show` output

**Issues reference**: #4 

---

:warning: Please, review it carefully.

### CODE REVIEW

File: tkgbot.py [hunk 1]
```diff
@@ -169,40 +169,87 @@ def __call__(self, chat_id, args):
...
+        line = ['  -'] # What are you looking at?
+
+        node_description = self._get_node_description(sub.node_id)
+        if node_description:
+            line.append(f' {node_description}')
+        elif sub.node.name and sub.node.name:
+            line.append(f' "{sub.node.name}"')
+
+        if not sub.exception:
+            if sub.no_replies or sub.no_comments:
```
Comment (vitaliyp) [inline tkgbot.py]: exception, no-replies and no-comments are already mutually excluding. This check is done upon submitting a subscription in such a way, that only one of these flags can be set at a time, so maybe there is not need to check it here.
See the `SubscribeCommand`.

**Comment by vitaliyp** [comment]
While this change improves the looks a little bit, it does close to nothing to improve usability. Here is a simple example:
The user is subscribed to two teams' subforums but doesn't want to receive notification about upcoming trainings for one of them. So he excludes training topic. /show now looks like this:
```
Team 1 [1234]
Team 2 [1235]

excluding
Trainings [321]
```
Now try to figure out to which section Trainings topic belong.

The cool solution would be to show subs as a tree:
```
Team 1 [1234]
└ X Trainings [321]
Team 2 [1235]
```
One problem i see with this solution is the possibility that the tree will get messed up when the chat window isn't wide enough, for example on mobile devices or when telegram windows is sized down.

Another option would be to just linearize the tree and put some separators between the sections (basically the same as the example above, but without drawing a tree).
```
Team 1 [1234]
X Trainings [321]

Team 2 [1235]
```

**Comment by dector** [comment]
Yup, the goal was to just improve existing output a bit. Not intended to implement #4.
I think that designing new hierarchical view requires more effort and discussions that where put here.

File: tkgbot.py (hunk 1 above)
Comment (dector) [reply tkgbot.py]: My suggestion is that I can't rely on things that are not reflected in code (or data structures).
So yes, technically this check is redundant but it makes code more resistant to possible future errors.

As an option - something like
`# TODO make subscription type as enum`
can be added to describe this line.

Comment (vitaliyp) [reply tkgbot.py]: This is a valid point.

File: locales/uk_UA/LC_MESSAGES/tkgbot.po [hunk 2]
```diff
@@ -75,24 +80,66 @@ msgstr ""
...
+"Спробуйте `/sub events`."
+
+#: tkgbot.py:241
+msgid "You *will receive* notifications from\n"
+msgstr "Ви *будете отримувати* повідомлення з\n"
+
+#: tkgbot.py:247
+msgid ""
+"\n"
+"*expluding*\n"
```
Comment (vitaliyp) [inline locales/uk_UA/LC_MESSAGES/tkgbot.po]: Dude, really?

**Pull request merged**